src/librustc_metadata/common.rs

```diff
@@ -247,7 +247,8 @@ pub const tag_rustc_version: usize = 0x10f;
 pub fn rustc_version() -> String {
     format!(
         "rustc {}",
-        option_env!("CFG_VERSION").unwrap_or("unknown version")
+//        option_env!("CFG_VERSION").unwrap_or("unknown version")
+        "nightly edition"
```

Comment:
No.

On Tue, May 17, 2016 at 3:34 PM, bluss <notifications@github.com> wrote:

> In src/librustc_metadata/common.rs
> <https://github.com/rust-lang/rust/pull/33491#discussion_r63513124>:
>
> > @@ -247,7 +247,8 @@ pub const tag_rustc_version: usize = 0x10f;
> >  pub fn rustc_version() -> String {
> >      format!(
> >          "rustc {}",
> > -        option_env!("CFG_VERSION").unwrap_or("unknown version")
> > +//        option_env!("CFG_VERSION").unwrap_or("unknown version")
> > +        "nightly edition"
>
> Is this an intentional change?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/33491/files/65ad935737138eb307fdd01279ba5553a047bb6c#r63513124>
>